static/src/javascripts/projects/common/modules/commercial/commercial-feature-policies.js

```diff
@@ -0,0 +1,114 @@
+define([
+    'common/utils/_',
+    'common/utils/location',
+    'common/utils/config',
+    'common/modules/commercial/user-ad-preference',
+    'common/modules/user-prefs'
+], function (
+    _,
+    location,
+    config,
+    userAdPreference,
+    userPrefs
+) {
+    var policies = {};
+
+    policies.defaultAds = function () {
+        return new CommercialFeatureSwitches(true);
+    };
+
+    policies.sensitiveContent = function () {
+        // Avoid inappropriate adverts on pages with content of a sensitive nature
+        if (config.page.shouldHideAdverts || config.page.section === 'childrens-books-site') {
+            return new CommercialFeatureSwitches(false);
+        }
+    };
+
+    policies.sslContent = function () {
+        if (config.page.isSSL && config.page.section !== 'admin') {
+            return new CommercialFeatureSwitches(false);
+        }
+    };
+
+    policies.noadsHash = function () {
+        if (location.getHash().match(/[#&]noads(&.*)?$/)) {
+            return new CommercialFeatureSwitches(false);
+        }
+    };
+
+    policies.userPrefs = function () {
+        if (userPrefs.isOff('adverts')) {
+            return new CommercialFeatureSwitches(false);
+        }
+    };
+
+    policies.adfreeExperience = function () {
+        if (userAdPreference.hideAds) {
+            return {
+                articleMPUs : false,
+                sliceAdverts : false,
+                popularContentMPU : false,
+                videoPreRolls : false
+            };
+        }
+    };
+
+    policies.identityPages = function () {
+        if (config.page.contentType === 'Identity' || config.page.section === 'identity') {
+            return {thirdPartyTags : false};
+        }
+    };
+
+    policies.nonArticlePages = function () {
+        if (config.page.contentType !== 'Article' && !config.page.isLiveBlog) {
```

Comment:
@kelvin-chappell - there seems to be one in the model code - it seems I could add this property to the Content class' metadata in content.scala. Would this be the appropriate place to do it?